app/routes/notification/notification.js

```diff
@@ -95,6 +95,11 @@ router.route('/')
     }
   })
   .post(async (req, res) => {
+    // TODO: Delete admin check once notification is properly implemented/tested
```

Comment:
can you make a ticket for this?

Reply:
https://www.bcgsc.ca/jira/browse/DEVSU-2082